GNUmakefile

```diff
@@ -23,6 +24,11 @@ clean:
 release:
 	@git tag v${NEXT_VERSION} && git push --mirror
 	@echo "Pushed v${NEXT_VERSION}"
+	GOPROXY=https://proxy.golang.org GO111MODULE=on go get github.com/jfrog/${BINARY_NAME}@v${NEXT_VERSION}
```

Comment:
I know I added this to the `artifactory` provider to refresh the https://pkg.go.dev whenever we make a release. However, it looks like this is unnecessary as pkg.go.dev seems to pick up all the new releases automatically (as it should have).

This should be omitted from the PR